src/crimson/osd/main.cc

```diff
@@ -193,6 +190,7 @@ static void override_seastar_opts(std::vector<const char*>& args)
     // with the deployment tools, like cephadm and rook, which don't set, for
     // instance, aio-max-nr for us. but we should fix this, once crimson is able
     // to run on a multi-core system, i.e., once m-to-n problem is resolved.
+    std::cout << "warn: added seastar option --smp 1" << std::endl;
```

Comment:
hmm, i think this is a known limit of existing crimson implementation at this moment, and we have already a TODO for it. do we really need to show warn our user when overriding this option?

@rzarzynski what do you think?

Reply:
This message will print whenever OSD adds the option `--smp 1` to seastar app, which was implicit if the first-time user doesn't have the chance to read the comment yet.

Reply:
i knew. and my question still holds.

Reply:
after a second thought, changing whatever use specifies explicitly is a surprising behavior. and the user should be aware of this. so LGTM.